pages/in/widget/pay/index.vue

```diff
@@ -714,8 +727,17 @@ export default {
         if (state) url.query.state = state;
         if (remarks) url.query.remarks = remarks;
         if (success !== undefined) url.query.success = success;
-        url.set('query', url.query);
-        window.location.href = url.toString();
+        if (this.opener) {
+          const message = JSON.stringify({
+            action: 'TX_FEE_SUBMITTED',
+            data: url.query,
+          });
+          this.windowOpener.postMessage(message, this.redirectOrigin);
+          window.close();
+        } else {
+          url.set('query', url.query);
+          window.location.href = url.toString();
```

Comment:
I think it is better to make sure `redirectUri` is well defined and is inside users' white list before redirecting

Reply:
Should we just separate it like iscn widget?

Reply:
or you can mark a flag in the asyncData block where we check the whitelist already (which this PR skipped)